src/DGtal/shapes/doc/moduleHalfEdgeMesh.dox

```diff
@@ -162,6 +162,32 @@ It is worthy to note the following elements in this representation:
 - given an arc (i.e. a couple of vertices), you can get the
   corresponding half-edge in logarithmic time (stored as a map).
 
+
+\subsection HEM_sec1_4 Modifying operations: flip, split, merge
+
+Since 1.1, there is a limited support for classical flip, split and
+merge operations. For now, it is limited to triangulated half-edge
+date structures, i.e. a face is bordered by three half-edges. 
+
+- HalfEdgeDataStructure::isFlippable tells if some edge is flippable,
+  i.e. it is bordered by two valid triangles;
+- HalfEdgeDataStructure::flip flips the given edge if the edge is
+  flippable, so that the edge now connects the two other vertices of
+  the bordering two triangles;
+- HalfEdgeDataStructure::split splits the given edge if the edge is
+  flippable, i.e. creates a new vertex on this edge and two new
+  triangles;
+- HalfEdgeDataStructure::isMergeable tells if some edge is mergeable,
+  i.e. (1) it is bordered by two triangles, (2) there is no boundary
+  vertex in the two triangles bordering the edge;
+- HalfEdgeDataStructure::merge merges the given edge and the two
+  bordering triangles, leaving only one vertex and two edges, only if
+  the edge is mergeable.
+
+There is also a method HalfEdgeDataStructure::isValid that performs a
+lot of checks on the validity of the data structure.
+
```

Comment:
add  something saying that all these operations are O(1)

Reply:
if they are

Reply:
well.. there are not for HDS.. but they are for Triangulated surfaces.

Reply:
I've updated the doc